boards/sparkfun/micromod/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+.. _boardname_linkname:
+
+SparkFun MicroMod board Processor
+#################################
+
+Overview
+********
+
+MicroMod is a solderless, modular interface ecosystem that uses the M.2 standard
+to mix and match your choice of processor with specific Function Boards or
+stand-alone Carrier Boards. A MicroMod processor board is approximately 22x22 mm,
+and can insert into any MicroMod carrier board. More information can be found in
+the `Micromod specification website`_.
+
+All Micromod board targets support the following hardware features:
+
+- USB host mode compliant to 2.0 specification
+- GPIO
+- 2 UART
+- 2 I2C
+- 2 SPI
+- 2 ADC
+- 2 PWM
+- Watchdog Timer (WDT)
+
+.. figure:: img/sparkfun_micromod.webp
+   :align: center
+   :alt: Sparkfun Micromod board
+
+   Sparkfun Micromod board (Credit: https://www.sparkfun.com)
+
+Zephyr currently supports the following SoCs:
+
+- ``micromod/nrf52840``
+
+SparkFun MicroMod nRF52840 board Processor
+##########################################
+
+Overview
+********
+
+The ``micromod/nrf52840`` board target features the nRF52840 SoC
+from Nordic Semiconductor, the SparkFun MicroMod nRF52840 Processor offers
+a powerful combination of ARM Cortex-M4 CPU and 2.4 GHz Bluetooth transceiver
+in the MicroMod form-factor. More information can be found in
+the `Micromod nRF52840 guide`_.
+
+Hardware
+********
+
+The ``micromod/nrf52840`` board target supports the following
+hardware features:
+
+- ARM Cortex-M4 CPU with floating point unit (FPU)
+   - 1MB internal Flash
+   - 256kB internal RAM
+- Integrated 2.4GHz radio with support for Bluetooth Low Energy (BLE) and ieee802154
```

Comment:
```suggestion
- ARM Cortex-M4 CPU with floating point unit (FPU)

   - 1MB internal Flash
   - 256kB internal RAM

- Integrated 2.4GHz radio with support for Bluetooth Low Energy (BLE) and ieee802154
```

renders as a "definition list" otherwise

Reply:
ok!